File: app/src/main/java/com/example/amconsofttest/io/realm_models/AddressRealm.java
package com.example.amconsofttest.io.realm_models;

import io.realm.RealmObject;

public class AddressRealm extends RealmObject {

    private String street;
    private String suite;
    private String city;
    private String zipcode;
    private GeoRealm geo;

    public String getStreet() {
        return street;
    }

    public AddressRealm setStreet(String street) {
        this.street = street;
        return this;
    }

    public String getSuite() {
        return suite;
    }

    public AddressRealm setSuite(String suite) {
        this.suite = suite;
        return this;
    }

    public String getCity() {
        return city;
    }

    public AddressRealm setCity(String city) {
        this.city = city;
        return this;
    }

    public String getZipcode() {
        return zipcode;
    }

    public AddressRealm setZipcode(String zipcode) {
        this.zipcode = zipcode;
        return this;
    }

    public GeoRealm getGeo() {
        return geo;
    }

    public AddressRealm setGeo(GeoRealm geo) {
        this.geo = geo;
        return this;
    }
}

File: app/src/main/java/com/example/amconsofttest/io/realm_models/AddressRealm.kt
package com.example.amconsofttest.io.realm_models

import io.realm.RealmObject

open class AddressRealm : RealmObject() {

    private var street: String? = null
    private var suite: String? = null
    private var city: String? = null
    private var zipcode: String? = null
    private var geo: GeoRealm? = null

    fun getStreet(): String? {
        return street
    }

    fun setStreet(street: String): AddressRealm {
        this.street = street
        return this
    }

    fun getSuite(): String? {
        return suite
    }

    fun setSuite(suite: String): AddressRealm {
        this.suite = suite
        return this
    }

    fun getCity(): String? {
        return city
    }

    fun setCity(city: String): AddressRealm {
        this.city = city
        return this
    }

    fun getZipcode(): String? {
        return zipcode
    }

    fun setZipcode(zipcode: String): AddressRealm {
        this.zipcode = zipcode
        return this
    }

    fun getGeo(): GeoRealm? {
        return geo
    }

    fun setGeo(geo: GeoRealm): AddressRealm {
        this.geo = geo
        return this
    }
}
